coalib/parsing/DefaultArgParser.py

```diff
@@ -249,6 +249,15 @@ def default_arg_parser(formatter_class=None):
         ' additional arguments. If used without arguments, all bears specified'
         ' with --bears will be debugged (even implicit dependency bears).')
 
+    outputs_group.add_argument(
+        '--profile', nargs='?', const=True,
+        help='Enable bear profiling with cProfile, that can help to create'
+        ' a Performant Bears by profiling the Bear code. To specify where to'
+        ' to dump the profile files, supply the path or the directory name, all'
+        ' the bear (even implicit dependency bears) in a section will get'
+        ' profile and there profile data will save in the specified directory'
```

Comment:
I would place a period here so it's easier to read:
"In a section will be profiled. Profile data will **be** saved in the specified directory or path."